library/package.json

```diff
@@ -22,6 +22,9 @@
     "barrel": "eslint --fix src/index.ts",
     "barrel:watch": "nodemon --delay 1 -e ts,tsx --watch src/hooks -x 'npm run barrel --silent || exit 1'",
```

Comment:
What does `barrel` mean?

Reply:
Barrel files are files that re-export exports of other files. (See https://github.com/ubilabs/google-maps-react-hooks/blob/develop/src/index.ts)

Reply:
Something new every day!